packages/styles/scss/patterns/listsection/_listsection.scss

```diff
@@ -0,0 +1,105 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection { 
+  .#{$prefix}--listsection {
+    &__container {
+      padding-left: $layout-01;
+      padding-right: $layout-01;
```

Comment:
@sannminwin I believe if you add `background: $ui-background;` here, the dark theme should work.

Reply:
Perfect, thank you

Reply:
Added theme background color by [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/573/commits/b90dd7ec11c2c1e80f979e2eda4c71ead47902ec)